website/docs/index.html.markdown

```diff
@@ -43,7 +43,7 @@ The following arguments are supported:
 * `enable_ddi` - (Optional) This sets the permission schema to a DDI-compatible schema. 
 Users of the managed SaaS product should not need to set this.
 Users of DDI should set this to true if managing teams, users, or API keys through this provider.
-* `rate_limit_parallelism` - (Optional) Integer for parallelism amount (terraform's default is 10).
+* `rate_limit_parallelism` - (Optional) Integer for alternative rate limit and parallelism strategy.
```

Comment:
Any reason to remove the parenthetical regarding the default value?

Reply:
Oh, perhaps not. I must have cut it out while copying James' suggested edit. Am I able to adjust my suggested changes or should I submit another PR?